charts/newrelic-infrastructure/values.yaml

```diff
@@ -62,10 +59,9 @@ kubelet:
     - operator: "Exists"
       effect: "NoExecute"
   nodeSelector: {}
-  affinity:
-    nodeAffinity: {}
-  extraEnv: []
-  extraEnvFrom: []
+  affinity: {}
+  env: []
+  envFrom: []
```

Comment:
As said above, I'm not fully in with this change.

Reply:
Paolo stated this https://github.com/newrelic/nri-kubernetes/pull/383#discussion_r845194876 and this was solved in [#64e304d](https://github.com/newrelic/nri-kubernetes/pull/383/commits/64e304d72084e4070dc15553499680bc4f1ba85a).